### PR TITLE
added new zaid's chickens profile pic and info

### PR DESCRIPTION
**Pull request by zaidmamsa1**

### CODE REVIEW

**Review by rrrliu — CHANGES_REQUESTED** [review]
LMAO excellent picture choice
1. don't check in .DS_Store
2. fix merge conflicts with `git pull origin/main` on your branch

**Review by leebianca — DISMISSED** [review]

File: node_modules/.yarn-integrity [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "systemParams": "darwin-x64-72",
+  "modulesFolders": [],
+  "flags": [],
+  "linkedModules": [],
+  "topLevelPatterns": [],
+  "lockfileEntries": {},
+  "files": [],
+  "artifacts": {}
+}
```
Comment (rrrliu) [inline node_modules/.yarn-integrity]: `git rm -r` the node_modules directly under cofed

File: yarn-error.log [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+npm manifest: 
+  No manifest
+
+yarn manifest: 
+  No manifest
+
+Lockfile: 
+  # THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+  # yarn lockfile v1
```
Comment (rrrliu) [inline yarn-error.log]: remove this

File: yarn.lock [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (rrrliu) [inline yarn.lock]: remove

**Review by rrrliu — CHANGES_REQUESTED** [review]
remove the yarn metadata under `cofed/` (remember to run everything under `cofed/client/`)

**Review by rrrliu — APPROVED** [review]
🥚 🐣 🐥 🐤 🐔 🍗

**Pull request merged**